libcob/java.c

```diff
@@ -0,0 +1,97 @@
+/*
+   Copyright (C) 2024 Free Software Foundation, Inc.
+   Written by Vedant Tewari
+   This file is part of GnuCOBOL.
+   The GnuCOBOL runtime library is free software: you can redistribute it
+   and/or modify it under the terms of the GNU Lesser General Public License
+   as published by the Free Software Foundation, either version 3 of the
+   License, or (at your option) any later version.
+   GnuCOBOL is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU Lesser General Public License for more details.
+   You should have received a copy of the GNU Lesser General Public License
+   along with GnuCOBOL.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include <jni.h>
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <common.h>
+
+static JavaVM *jvm = NULL;
+static JNIEnv *env = NULL;
+
+typedef struct __cob_java_static_method {
+	jclass cls;
+	jmethodID mid;
+} cob_java_handle;
+
+static int			/* non-zero means there's an error */
+cob_java_initialize() {
+    /* JDK/JRE 6 VM initialization arguments */
+    JavaVMInitArgs args;
+    JavaVMOption* options = (JavaVMOption*)cob_malloc(sizeof(JavaVMOption) * 1);
+    args.version = JNI_VERSION_1_6;
+    /* inline */
+    args.nOptions = 1;
+    options[0].optionString = cob_malloc(strlen("-Djava.class.path=/usr/lib/java") + 1);
+    strcpy(options[0].optionString, "-Djava.class.path=/usr/lib/java");
```

Comment:
This is a bit weird. In any case, I guess this should be given via an environment variable (`CLASSPATH`)? Maybe setting this `optionString` field is not needed at all.